resources/assets/js/annotations/components/annotationCanvas/magicWandInteraction.vue

```diff
@@ -60,30 +60,30 @@ export default {
             }
         }
     },
+    watch: {
+        isMagicWanding(isMagicWanding) {
+            this.toggleMagicWandInteraction(isMagicWanding);
+        },
+    },
     created() {
-        if (this.canAdd) {
-            Keyboard.on('Shift+g', this.toggleMagicWand, 0, this.listenerSet);
-            this.$watch('image', this.maybeUpdateMagicWandSnapshot);
-            this.$watch('image', this.maybeSetMagicWandLayer);
-            this.$watch('isMagicWanding', this.toggleMagicWandInteraction);
-        }
+        this.$watch('image', this.maybeUpdateMagicWandSnapshot);
+        this.$watch('image', this.maybeSetMagicWandLayer);
     },
     mounted() {
         // Initialize the magic wand interaction here because we have to wait for
         // the non-reactive properties of annotationCanvas to be initialized.
-        if (this.canAdd) {
-            magicWandInteraction = new MagicWandInteraction({
-                map: this.map,
-                source: this.annotationSource,
-                style: Styles.editing,
-                indicatorPointStyle: Styles.editing,
-                indicatorCrossStyle: Styles.cross,
-                simplifyTolerant: 0.1,
-            });
-            magicWandInteraction.on('drawend', this.handleNewFeature);
-            magicWandInteraction.setActive(false);
-            this.map.addInteraction(magicWandInteraction);
-        }
+        magicWandInteraction = new MagicWandInteraction({
+            map: this.map,
+            source: this.annotationSource,
+            style: Styles.editing,
+            indicatorPointStyle: Styles.editing,
+            indicatorCrossStyle: Styles.cross,
+            simplifyTolerant: 0.1,
+        });
+        magicWandInteraction.on('drawend', this.handleNewFeature);
+        magicWandInteraction.setActive(false);
+        this.map.addInteraction(magicWandInteraction);
+        Keyboard.on('Shift+g', this.toggleMagicWand, 0, this.listenerSet);
```

Comment:
This line should be moved back to `created()`.

Reply:
You mean line 86? Otherwise magicWandInteraction is not initialized yet and an exception will be thrown.

Reply:
Yes, only the single line.